src/lib/mongoBackend/mongoSubscribeContext.cpp

```diff
@@ -202,7 +203,7 @@ HttpStatusCode mongoSubscribeContext
     }
 
     /* Adding format to use in notifications */
-    sub.append(CSUB_FORMAT, "JSON");
+    sub.append(CSUB_FORMAT, notificationFormatToString(requestP->attrsFormat));
```

Comment:
However, I think I have see code for setting attrsFormat *after* the parsing stage (in the serviceRoutines). I have lo take a closer look.